samples/net/sockets/sntp_client/src/main.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_sntp_client_sample, LOG_LEVEL_DBG);
+
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <net/socket.h>
+#include <net/net_ip.h>
+#include <net/sntp.h>
+
+#define SNTP_PORT 123
+
+void resp_callback(struct sntp_ctx *ctx, int status,
+		   u64_t epoch_time)
+{
+	LOG_DBG("time: %lld", epoch_time);
```

Comment:
AFAIK, this will not work. Only `printk()` can print 64 bit numbers. Using `LOG_DBG()` this could be done the following way, although it looks a bit hacky and prints the number as a hex value:
```c
LOG_DBG("time: %08x%08x", (u32_t) epoch_time >> 32, (u32_t) epoch_time & 0xffffffff);
```
Maybe there is some nicer way to print it as a decimal.

Reply:
If logger doesn't support these kind of formats. May be open a issue to fix the problem in logger. For time being one can hack their samples. I will keep it as is.

Reply:
This is a known limitation of the logging system and doubt it will be changed soon. See e.g. issue #12098. I may be wrong though. @nashif Any thoughts on this?

Reply:
It might be a limitation atm, but to me this sounds like a serious bug if 64-bit values cannot be printed and should be fixed in the logger instead of adding some workarounds. Do you know if there is an issue created for this already?

Reply:
Btw, the #12098 was for minimal libc, the issue is not related to logger.